ruby/basics/variables_strings_and_numbers.md

```diff
@@ -1,119 +1,371 @@
+
+The most basic thing you can do to write clean, maintainable code is name your variables properly.
+
+variable names should always be lowercase and multiple words that make up a variable name should be split by an underscore this is known as _snake_case_.
+
+```(ruby)
+#bad
+a = "Hello"
+number = 2
+
+#good
+age = 19
+name = "John"
+can_swim = false
 ```
-Here, we're assigning the string `'John'` to the variable `name` and the value 
-`34` to the variable `age`
 
-### Different ways to name them
-- naming conventions
-- description
-- code snippets
-- further reading(if any)
+### Assignment
+1. Read the [variables chapter](http://ruby.bastardsbook.com/chapters/variables/) from the Bastards book of Ruby for alternative metaphors and deeper topics about variables. Don't worry if you don't understand everything completely, especially the sections about references. This will become more intuitive as you use ruby by completing exercises and projects.
+2. Open up a [Ruby Repl](https://repl.it/languages/ruby) or IRB in your command line and try naming some variables and assigning some value to it. Don't try at this stage to name good variables but instead experiment with different variable names, run the Repl and see if it's valid. Try using symbols or numbers in your variable names. See what works and what doesn't. If you come across anything quirky Google it to find out why.
+
 
 ## Strings
+Strings, strings, wonderful things, use them well and your app will grow wings. Or something.
+
+At first glance, you'd be forgiven for thinking Strings are just a bunch of characters that aren't very useful beyond getting user input and outputting some information to the screen, but like Burt Reynolds passing up the chance of playing Han Solo, you'd be wrong. Very wrong. What were you thinking Burt?
+
+From the prep course you should have a good understanding of Strings so we won't bore you to death recapping what you already know. Instead we'll just cover some of the pitfalls and more interesting features and let the reading assignments do the rest.
+
 - double and single quotes in strings
--  
+Strings can be formed with either double `""` or single`''` quotes. They are pretty similar but there are some differences.
+
+### Interpolation
+Use double quotes for String interpolation.
+
+```(ruby)
+name = "Odin"
+puts "Hello, #{name}" # => "Hello, Odin"
+puts 'Hello, #{name}' # => "Hello, #{name}"
+```
 
 ### Concatenation
-- using the shovel operator or the += method
-- concatenating variables
+In true Ruby style. There are plenty of ways to concatentate with Ruby
 
-### Interpolation
-- using variables inside strings
-- using expressions inside strings
+```(ruby)
+# with the plus operator
+"Welcome " + "to " + "Odin!" # => "Welcome to Odin!"
+
+# with the shovel operator
+"Welcome " << "to " << "Odin!" # "Welcome to Odin!"
+
+# with the concat method
+"Welcome ".concat("to ").contact("Odin!") # => "welcome to Odin!"
```

Comment:
# => "Welcome to Odin!" 

In the return value, 'welcome' should be capitalized